cli/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ _Released 7/16/2024_
 - Replaced `json-lint` with `json-parse-even-better-errors`. This removes the [CVE-2021-23358](https://nvd.nist.gov/vuln/detail/CVE-2021-23358) vulnerability being reported in security scans. Addresses [#28207](https://github.com/cypress-io/cypress/issues/28207).
 - Updated `minimatch` from `3.0.4` to `3.1.2`. Addressed in [#29821](https://github.com/cypress-io/cypress/pull/29821).
 
+**Misc:**
+
+- Added `CYPRESS_NO_VERIFY` flag to enable suppression of Cypress verification checks. Addresses [#22243](https://github.com/cypress-io/cypress/issues/22243)
```

Comment:
`CYPRESS_NO_VERIFY` is not the name of the environment variable in the code, which is instead `CYPRESS_SKIP_VERIFY`